embed-api-v3.rst

```diff
@@ -0,0 +1,170 @@
+resulting in a lot of friction in the form of context switching.
+
+One way of mitigating the problem is to show a small preview
+in the form of tooltip or modal with the contents of the link.
+This is where the Embed API and sphinx-hoverxref come in.
+
+`sphinx-hoverxref`_ is a Sphinx extension that addresses this problem
+by adding tooltips on the cross references of the HTML documentation
+with the content of the linked section.
+To do so, it inserts a small JavaScript file that
+calls the Read the Docs Embed API when the user hovers any link.
+:doc:`Back in June </hoverxref-intersphinx>` we announced
+intersphinx support in `sphinx-hoverxref`_ 0.6b1,
+which allowed users to include modals and tooltips
+also for cross references to other projects.
+
+.. figure:: img/sphinx-hoverxref-showcase.gif
+   :width: 60%
+   :align: center
+   :alt: sphinx-hoverxref displaying tooltips for cross references
+   :target: /_images/sphinx-hoverxref-showcase.gif
+
+   sphinx-hoverxref displaying a tooltip including an equation
+
+.. _sphinx-hoverxref: https://sphinx-hoverxref.readthedocs.io/
+
+Some big projects already using it include `Tweepy <https://docs.tweepy.org/>`_,
+a Python client for Twitter, and `Scrapy <https://docs.scrapy.org/>`_,
+a framework for crawling websites.
+
+Embed API v3 and sphinx-hoverxref 1.0
+-------------------------------------
+
+After some months of work, **we are excited to publish v3 of our Embed API,
+and with it, version 1.0 of sphinx-hoverxref**.
+
+Among other things,
+the new versions allow you to
+:ref:`embed content from pages hosted outside Read the Docs <guides/embedding-content:embedding content from your documentation>`.
+For security reasons, we have an allowlist of such external domains
+that currently includes Python, NumPy, SciPy, SymPy,
+and we would like to invite the community to suggest more domains to add.
+
+Other additions and fixes include:
+
+- Support for JS and CSS assets in Sphinx 4.1+.
+- Support for ``glossary`` / ``term`` and sphinxcontrib-bibtex.
+- Avoid showing the browser built-in tooltip for links that have the extension enabled.
+
+Finally, version 3 of the Embed API
+paves the way for supporting non-Sphinx projects in the future.
+
+Embedding content with sphinx-hoverxref
+---------------------------------------
+
+To use sphinx-hoverxref in your Read the Docs project,
+:doc:`declare it as part of your dependencies <guides/reproducible-builds>`:
+
+.. code-block::
+   :caption: requirements.txt
+   :emphasize-lines: 3
+
+   sphinx==4.3.1
+   sphinx_rtd_theme==1.0.0
+   sphinx-hoverxref==1.0.0
+
+And add it to your list of Sphinx extensions:
+
+.. code-block::
+   :caption: conf.py
+   :emphasize-lines: 3
+
+   extensions = [
+       # ...other extensions
+       "hoverxref.extension",
+   ]
+
+To enable the extension on all ``:ref:`` of your documentation,
+set the ``hoverxref_auto_ref`` to ``True``:
+
+.. code-block::
+   :caption: conf.py
+
+   hoverxref_auto_ref = True
```

Comment:
I kinda hate that this is required. Is this not a reasonable default?

Reply:
Let's continue the conversation at https://github.com/readthedocs/sphinx-hoverxref/issues/163. For sphinx-hoverxref 1.0.0, this is needed.